datacapture/src/main/java/com/google/android/fhir/datacapture/QuestionnaireViewModel.kt

```diff
@@ -198,6 +197,9 @@ internal class QuestionnaireViewModel(state: SavedStateHandle) : ViewModel() {
               )
             }
           if (enabled) {
```

Comment:
Early return to make the code more readable:

```suggestion
          if (!enabled) {
            questionnaireResponseItem.removeAnswers()
            emptyList()
          }
```

Reply:
Please check, this is what you meant.